api/app/responses/api.py

```diff
@@ -106,6 +106,12 @@ def post(self):
                 user = user_repository.get_by_id(user_id)
                 response = response_repository.get_by_id_and_user_id(response.id, user_id)
                 self.send_confirmation(user, response)
+                
+                event = event_repository.get_event_by_response_id(response.id)
+                if event.event_type == EventType.CONTINUOUS_JOURNAL:
+                    all_admin_users = event_repository.get_all_admin(event_id=response.id)
+                    for admin_user in all_admin_users:
+                        self.send_confirmation(admin_user, response, admin_user=True)
```

Comment:
The `send_confirmation` method takes in a user object which we luckily have access to from an `EventRole`
```python
for admin_event_role in all_admin_event_roles:
    self.send_confirmation(admin_event_role.user, response, admin_user=True)
```